### ISSUE TITLE
Separate the standard PackageJson types into it's own type  

### ISSUE THREAD

**Issue by barak007** [sindresorhus/type-fest#122]
It will be nice to separate the standard part from the `PackageJson` type into it's own type let's say `PackageJson.PackageJsonStandard`.
This will allow to use `keyof PackageJson.PackageJsonStandard` and get the actual keys. 
Right now since there is ` & {[key: string]: unknown;}` at the end of the type the `keyof` operator is not very useful it returns `string | number`.

My use-case is that I want to write a function to copy standard values from one package.json to anther and I want it to be typesafe.

Example: 
```js
function copyCommonPackageValues(
  target: PackageJson,
  origin: PackageJson,
  // This is not safe right now I need something like keyof PackageJson.PackageJsonStandard
  keys: Array<keyof PackageJson>
) {
  for (const key of keys) {
    const value = origin[key];
    if (value) {
      target[key] = value;
    }
  }
}
```

**Comment by sindresorhus** [comment]
I'm actually not sure why we have `& {[key: string]: unknown;}` there. Unless you see a good reason it's there, maybe we could just remove it? People that need it, could easily add that themselves.

**Comment by papb** [comment]
I agree, I wouldn't mind seeing this `& {[key: string]: unknown;}` removed as well.

**Comment by barak007** [comment]
Removing it will probably will be a major change for some people. If you have no problem with that I'm in. if not I can open a PR with the proposed change.

**Comment by sindresorhus** [comment]
Yes, PR welcome.
